twisted/conch/ssh/keys.py

```diff
@@ -39,6 +42,47 @@
 from twisted.python.deprecate import deprecated, getDeprecationWarningString
 from twisted.python.versions import Version
 
+# Curve lookup table
+curveTable = {
```

Comment:
My programming teacher told me to NEVER access variables outside of a class directly.  He was very strict about this, and and for good reason.

But this is python, and normal rules no longer apply. :)